docs/troubleshooting.mdx

```diff
@@ -1,26 +1,26 @@
 ---
 id: troubleshooting
+slug: /troubleshooting
 title: Troubleshooting
+description: Guide to troubleshooting some known errors and contacting the CCV with questions

```

Comment:
Will all the users know what CCV is? (i.e., is this only used by Brown researchers?) If not, add a link to CCV website.

Reply:
Also need to add a section at the end about contacting CCV as a last resort?